ci/releases.yml

```diff
@@ -48,7 +48,9 @@ suite-desktop github release:
       - codesign
   when: manual
   script:
-    - release-it --ci --config=ci/release-it.yml --disable-metrics
+    - gh config set prompt disabled
+    - echo $GITHUB_CLI_RELEASE_TOKEN | gh auth login --with-token
+    - gh release create --repo trezor/trezor-suite --draft v$(jq -r .version packages/suite-desktop/package.json)  --title "v$(jq -r .version packages/suite-desktop/package.json)" ./Trezor-Suite* latest*
```

Comment:
Just a tiny thing but I would prefer:

```
- VERSION=$(jq -r .version packages/suite-desktop/package.json)
- gh release create --repo trezor/trezor-suite --draft v${VERSION}  --title "v${VERSION}"  ./Trezor-Suite* latest*    
```

Reply:
Ok, let's do it this way :) updated the pr with this.